services/static-webserver/client/source/class/osparc/data/model/Wallet.js

```diff
@@ -22,14 +22,14 @@ qx.Class.define("osparc.data.model.Wallet", {
     this.base(arguments);
 
     this.set({
-      walletId: walletData["id"],
+      walletId: walletData["wallet_id"],
       name: walletData["name"],
       description: walletData["description"] ? walletData["description"] : null,
       thumbnail: walletData["thumbnail"] ? walletData["thumbnail"] : null,
-      walletType: walletData["type"] ? walletData["type"] : "personal",
-      accessRights: walletData["accessRights"],
-      credits: walletData["credits"] && walletData["credits"]["left"] ? walletData["credits"]["left"] : 0,
-      active: walletData["active"] ? walletData["active"] : false
+      owner: walletData["owner"] ? walletData["owner"] : null,
+      status: walletData["status"] ? walletData["status"] : false,
+      creditsAvailable: walletData["available_credits"] ? walletData["available_credits"] : 20,
```

Comment:
not weird to have 20 credits when none available?

Reply:
I would have this in the backend do avoid "hackers" which can give themselves more credits